dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/query/operators/TokenOperator.java

```diff
@@ -74,10 +73,8 @@ public Criterion getHibernateCriterion( QueryPath queryPath )
     }
 
     @Override
-    @SuppressWarnings( "unchecked" )
     public boolean test( Object value )
     {
-        String targetValue = caseSensitive ? getValue( String.class ) : getValue( String.class ).toLowerCase();
```

Comment:
FYI: case sensitivity is already handled within the called `test` method and does not to be handled here as well.